src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  gap: $gap;
+
+  @each $day, $mod in $start-day {
+    &--start-day--#{$day} :first-child {
+      margin-left: $mod;
+    }
+  }
+
+  @each $month, $value in $month-length-lib {
+    &--month-length--#{$month} :nth-last-child(-n + #{$value}) {
+      display: none;
+    }
+  }
+
+  &__cell {
+    box-sizing: border-box;
+    display: flex;
+    width: $cell-width;
+    height: $cell-height;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid black;
+    background-color: #eee;
+    transition-duration: 0.5s;
```

Comment:
Always add transition property. By default, transition property is all, and it's hard for the browser to control all properties